nextflow.config

```diff
@@ -224,10 +224,7 @@ profiles {
 
 // Load nf-core custom profiles from different Institutions
 includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/nfcore_custom.config" : "/dev/null"
-
-// Load nf-core/scrnaseq custom profiles from different institutions.
-// TODO nf-core: Optionally, you can add a pipeline-specific nf-core config at https://github.com/nf-core/configs
-// includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/pipeline/scrnaseq.config" : "/dev/null"
+includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/pipeline/scrnaseq.config" : "/dev/null"
```

Comment:
sneaky version bump addition 😉

Reply:
sneaky update of the nf-core tools linter :P
That check didn't fail last week ;)